invenio_mail/__init__.py

```diff
@@ -52,6 +52,51 @@
 Content-Type: text/plain; charset="utf-8"...
 
 
+Using the API
+-------------
+
+A nice API let you create a message from a template, so you just have to
+give the rights arguments to get the full message. Moreover, it creates
+a complete e-mail with HTML message and text only message.
+
+To do so, you need to instantiate a TemplatedMessage object, just like you
+would with a standard ``flask_mail.Message``:
+
+.. code-block:: python
+
+    from invenio_mail.api import TemplatedMessage
+    msg = TemplatedMessage(template_html="invenio_mail/base.html",
```

Comment:
Can check how the generated documentation looks like. Usually large indents doesn't render nicely, so something like might be better:

```python
msg = TemplatedMessage(
    template=…
    template_body=...
```